web_external/views/body/SubmitView.js

```diff
@@ -72,6 +84,9 @@ var SubmitView = View.extend({
         this.listenTo(this.uploadWidget, 'g:filesChanged', this.filesSelected);
         this.listenTo(this.uploadWidget, 'g:uploadStarted', this.uploadStarted);
         this.listenTo(this.uploadWidget, 'g:uploadFinished', this.uploadFinished);
+        this.$('.c-submission-approach-input').typeahead({
```

Comment:
According to the [documentation](https://github.com/bassjobsen/Bootstrap-3-Typeahead#via-javascript) this installs event handlers. To avoid retaining references to the DOM element when this view is destroyed, is it necessary to override `destroy` and call `.typeahead('destroy')`?

Reply:
Done